app/templates/components/article-meta.hbs

```diff
@@ -10,10 +10,9 @@
         <i class="ion-trash-a"></i> &nbsp; Delete Article
       </button>
     {{else}}
-      <button class="btn btn-sm {{if article.author.following "btn-secondary" "btn-outline-secondary"}}"
-        data-test-follow-author-button {{action onFollowAuthor article.author}}>
-        <i class="ion-plus-round"></i> &nbsp; {{if article.author.following "Unfollow" "Follow"}} {{article.author.id}}
-      </button>
+      {{#follow-user-button class="btn btn-sm" followAction=(action onFollowAuthor article.author)}}
```

Comment:
There are `classNameBindings` for `following` but it’s not being used. Can you pass the property to the component?